test/dart3_suggestors/null_safety_prep/required_flux_props_test.dart

```diff
@@ -0,0 +1,1160 @@
+              expectedPatchCount: 1,
+              input: withFluxComponentUsage('''
+                main() {
+                  final theStore = FooStore();
+                  ${maybeInvokeBuilder('''
+                  Foo()
+                    ..store = theStore
+                  ''')}
+                }
+              '''),
+              expectedOutput: withFluxComponentUsage('''
+                main() {
+                  final theStore = FooStore();
+                  ${maybeInvokeBuilder('''
+                  Foo()
+                    ..actions = null
+                    ..store = theStore
+                  ''')}
+                }
+              '''),
+            );
+          });
+
+          group('when a top-level actions var is available', () {
+            test('unless the type does not match (uses null instead)',
+                () async {
+              await testSuggestor(
+                isExpectedError: (err) => err.message.contains('theActions'),
+                expectedPatchCount: 1,
+                input: withFluxComponentUsage('''
+                  final theActions = BazFooActions();
+                  main() {
+                    final theStore = FooStore();
+                    ${maybeInvokeBuilder('''
+                    Foo()
+                      ..store = theStore
+                    ''')}
+                  }
+                '''),
+                expectedOutput: withFluxComponentUsage('''
+                  final theActions = BazFooActions();
+                  main() {
+                    final theStore = FooStore();
+                    ${maybeInvokeBuilder('''
+                    Foo()
+                      ..actions = null
+                      ..store = theStore
+                    ''')}
+                  }
+                '''),
+              );
+            });
+
+            test('and the type matches', () async {
+              await testSuggestor(
+                isExpectedError: (err) => err.message.contains('theActions'),
+                expectedPatchCount: 1,
+                input: withFluxComponentUsage('''
+                  final theActions = FooActions();
+                  main() {
+                    final theStore = FooStore();
+                    ${maybeInvokeBuilder('''
+                    Foo()
+                      ..store = theStore
+                    ''')}
+                  }
+                '''),
+                expectedOutput: withFluxComponentUsage('''
+                  final theActions = FooActions();
+                  main() {
+                    final theStore = FooStore();
+                    ${maybeInvokeBuilder('''
+                    Foo()
+                      ..actions = theActions
+                      ..store = theStore
+                    ''')}
+                  }
+                '''),
+              );
+            });
+          });
+
+          group('when an actions var is available in block function scope', () {
```

Comment:
(With the updates I suggested) I think it'd be good to add a couple test cases for function parameters and class fields.

Reply:
Added in https://github.com/Workiva/over_react_codemod/pull/268/commits/aea20f00a1cad1436a176e585a6341cc64c48800